### PR TITLE
added Go lang to courses-th.md

### PR DESCRIPTION
**Pull request by nostprumprajee**
> ## What does this PR do?
> Add article Go lang from medium
> 
> ## For resources
> ### Description
> ### Why is this valuable (or not)?
> This PR adds 6 article about Go lang for Beginner.
> 
> ### How do we know it's really free?
> Resources are free on Medium.
> 
> ### For book lists, is it a book? For course lists, is it a course? etc.
> Yes, for the article and interactive tutorial.
> 
> ## Checklist:
> * [x]  Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
> * [x]  Search for duplicates.
> * [x]  Include author(s) and platform where appropriate.
> * [x]  Put lists in alphabetical order, correct spacing.
> * [x]  Add needed indications (PDF, access notes, under construction)
> 
> ## Follow-up
> * Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
if it's a book, add it to the book list. Use markdown list syntax. 
Please read CONTRIBUTING.md guidelines!

**Comment by nostprumprajee** [comment]
Sorry my fault.
Resubmit Done.

**Comment by nostprumprajee** [comment]
> if it's a book, add it to the book list. Use markdown list syntax. Please read CONTRIBUTING.md guidelines!

Resubmit done.

**Comment by nostprumprajee** [comment]
> Is this 4 books, or one book? I'm confused.

one book sir.

**Comment by nostprumprajee** [comment]
> > > Is this 4 books, or one book? I'm confused.
> > 
> > 
> > one book sir.
> 
> Then better use first link, I've thought
> 
> ℹ️ There are related links inside as navigation. The first have the second and so on...

Ok, I'll fix them.

File: courses/free-courses-th.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * [C#](#c-sharp)
 * [C++](#cpp)
 * [Git](#git)
+* [Go](#go)
```
Comment (davorpa) [inline courses/free-courses-th.md:7]: Anchor doesn't work. Remove since you move to books :wink:

File: books/free-programming-books-th.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+* [Go](#go)
 * [IoT (internet of things)](#iot-internet-of-things)
 * [Java](#java)
 * [Python](#python)
 
 
+### Go
+
+* [ภาษา Go ตอน 1 ติดตั้ง และ Run Hello World](https://medium.com/odds-team/%E0%B8%AA%E0%B8%A3%E0%B8%B8%E0%B8%9B%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B9%80%E0%B8%A3%E0%B8%B5%E0%B8%A2%E0%B8%99%E0%B8%9E%E0%B8%B7%E0%B9%89%E0%B8%99%E0%B8%90%E0%B8%B2%E0%B8%99%E0%B8%A0%E0%B8%B2%E0%B8%A9%E0%B8%B2-go-%E0%B9%81%E0%B8%9A%E0%B8%9A-step-by-step-%E0%B8%88%E0%B8%B2%E0%B8%81-course-pre-ultimate-go-by-p-yod-%E0%B8%95%E0%B8%AD%E0%B8%99-1-%E0%B8%95%E0%B8%B4%E0%B8%94%E0%B8%95%E0%B8%B1%E0%B9%89%E0%B8%87-%E0%B9%81%E0%B8%A5%E0%B8%B0-d9ac7913e9a4) - 
+Chaiyarin Niamsuwan
```
Comment (eshellman) [inline books/free-programming-books-th.md:12]: needs to be on same line as link

**Review by davorpa — APPROVED** [review]

**Pull request merged**